gitlabform/gitlabform/core.py

```diff
@@ -172,6 +183,12 @@ def get_version(self, skip_version_check):
 
         return version
 
+    def terminate_error(self, terminate_after_error):
```

Comment:
I totally understand more now. At first I wasn't sure because I was having issues running and testing the program. I definitely see where I need to add the logic for terminating after the first error.